rasa/cli/scaffold.py

```diff
@@ -183,7 +183,7 @@ def run(args: argparse.Namespace) -> None:
         .ask()
     )
 
-    if not os.path.isdir(path):
+    if path is not None and not os.path.isdir(path):
```

Comment:
```suggestion
    if path and not os.path.isdir(path):
```
Currently, `if path is not None` means that if you provide an empty string as a path, it will allow to you (attempt to) create it:
```
? Please enter a path where the project will be created [default: current directory]
? Path '' does not exist 🧐. Create path?  Yes
Traceback (most recent call last):
  File "/Users/ella/miniconda3/envs/rasa/bin/rasa", line 11, in <module>
    load_entry_point('rasa', 'console_scripts', 'rasa')()
  File "/Users/ella/rasa/rasa/__main__.py", line 76, in main
    cmdline_arguments.func(cmdline_arguments)
  File "/Users/ella/rasa/rasa/cli/scaffold.py", line 187, in run
    _ask_create_path(path)
  File "/Users/ella/rasa/rasa/cli/scaffold.py", line 140, in _ask_create_path
    os.makedirs(path)
  File "/Users/ella/miniconda3/envs/rasa/lib/python3.7/os.py", line 221, in makedirs
    mkdir(name, mode)
FileNotFoundError: [Errno 2] No such file or directory: ''
```

Changing it to my suggestion will avoid this, as when defining `path = ''`, `if path` evaluates to `False`.